### PR TITLE
cli: implement /_testing endpoint for minting tokens

### PR DESCRIPTION
**Pull request by andrevmatos**
Based on top of #1910 
Part of #1692 

**Short description**
Implements minting tokens from the CLI API.

**Definition of Done**

- [x] Steps to manually test the change have been documented
- [x] Acceptance criteria are met
- [ ] Change has been manually tested by the reviewer (dApp) 

**Steps to manually test the change (dApp)**

1.
2.

### CODE REVIEW

File: raiden-cli/src/routes/testing.ts [hunk 1]
```diff
@@ -1,12 +1,30 @@
...
+async function mintTokens(this: Cli, request: Request, response: Response, next: NextFunction) {
+  try {
+    response.json({
+      transaction_hash: await this.raiden.mint(request.params.tokenAddress, request.body.value, {
+        to: request.body.to,
+      }),
+    });
+  } catch (error) {
+    if ([ErrorCodes.DTA_INVALID_ADDRESS, ErrorCodes.DTA_INVALID_AMOUNT].includes(error.message))
+      response.status(400).send('Invalid "value" or "to" fields.');
```
Comment (weilbith) [inline raiden-cli/src/routes/testing.ts:15]: Usually we directly forward the error message here. So this if-else is basically in first place to determine the error code to use. Else we should avoid to attempt introducing new error messages and simply use the probably way better messages by the SDK.

File: raiden-cli/src/routes/testing.ts [hunk 2]
```diff
@@ -1,12 +1,30 @@
...
+import { validateAddressParameter } from '../utils/validation';
 import { Cli } from '../types';
 
+async function mintTokens(this: Cli, request: Request, response: Response, next: NextFunction) {
+  try {
+    response.json({
+      transaction_hash: await this.raiden.mint(request.params.tokenAddress, request.body.value, {
+        to: request.body.to,
+      }),
+    });
```
Comment (weilbith) [inline raiden-cli/src/routes/testing.ts:12]: Nitpick: For me the idea of this endpoint is to mint the tokens (`POST` as active request) and then return the result. This is how my brain works: declarative. So I would first await the minting and then respond. Makes it easy to follow the code. Personally I would even "construct" the response object separately but I wouldn't push for it.

Comment (andrevmatos) [reply raiden-cli/src/routes/testing.ts:12]: Done.

File: raiden-cli/src/routes/testing.ts (hunk 1 above)
Comment (andrevmatos) [reply raiden-cli/src/routes/testing.ts:15]: Done.

**Pull request merged**